src/Models/ExternalBillingItem.php

```
@@ -0,0 +1,26 @@
+	public $registeredDate;
+	public $deregisteredDate;
+	public $billingText;
+	public $discountPercentage;
+	public $discountExpiry;
+	public $externalId;
+	public $agreementNumber;
+	public $pricePerMonth;
+	public $pricePerMonthExpiry;
+	public $pricePerMonthText;
+	public $quantity;
+}
```

PHPMD (2.15.0)

[warning] The class ExternalBillingItem has 16 fields. Consider redesigning ExternalBillingItem to keep the number of fields under 15. (undefined)

(TooManyFields)